.env.example

```diff
@@ -4,6 +4,7 @@ GRAPHQL_PROXY_PORT=4000
 GRAPHQL_PROXY_ENGINE_API_KEY=tapahtumat
 GRAPHQL_PROXY_CMS_API_BASE_URL=https://cms.test.kuva.hel.ninja/api/
 GRAPHQL_PROXY_LINKED_EVENTS_API_BASE_URL=https://api.hel.fi/linkedevents/v1/
```

Comment:
If you are using baseBaths object mapping approach this should be GRAPHQL_PROXY_LINKED_EVENTS_API_BASE_URL=https://api.hel.fi/ ?

Maybe GRAPHQL_PROXY_LINKED_EVENTS_API_BASE_URL is not the best name for this anymore because it's common base url for all City of Helsinki APIs?

Also remember to change this url to .gitlab-ci.yml file as well